app/routers/system.py

```diff
@@ -115,8 +119,10 @@ async def hw_info_sub(request: Request):
     summary="Reboots the system",
     dependencies=[Depends(JWTBearer())],
 )
-def reboot_system():
-    return HTTPException(status.HTTP_501_NOT_IMPLEMENTED)
+async def reboot_system():
+    loop = asyncio.get_event_loop()
+    loop.create_task(shutdown(True))
+    return
```

Comment:
I'm not sure we have to create a task here. I will try this. Also an empty return shouldn't be necessary.

Reply:
I've tried this today. We mustn't create a task here. The function definition could be as follows:
```python
async def reboot_system() -> bool:
    await shutdown(True)
    return True
```
I think it'd be good practice to return true if the script ran without errors.